jerry-core/ecma/builtin-objects/ecma-builtin-global.cpp

```diff
@@ -615,7 +620,7 @@ ecma_builtin_global_object_decode_uri_helper (ecma_value_t uri __attr_unused___,
 
     input_char_p += 3;
```

Comment:
Could you, please, define this constant:
`#define URI_ENCODED_BYTE_SIZE (3)`
and replace all occurences of `+= 3`.